infra/scoped/api-documentation.tf

```diff
@@ -0,0 +1,585 @@
+resource "aws_api_gateway_documentation_version" "v1" {
+  rest_api_id = aws_api_gateway_rest_api.identity.id
+  version     = "v1"
+
+  depends_on = [
+    aws_api_gateway_documentation_part.api,
+    aws_api_gateway_documentation_part.auth,
+    aws_api_gateway_documentation_part.auth_post,
+    aws_api_gateway_documentation_part.users,
+    aws_api_gateway_documentation_part.users_get,
+    aws_api_gateway_documentation_part.users_post,
+    aws_api_gateway_documentation_part.users_userid,
+    aws_api_gateway_documentation_part.users_userid_get,
+    aws_api_gateway_documentation_part.users_userid_put,
+    aws_api_gateway_documentation_part.users_userid_delete,
+    aws_api_gateway_documentation_part.users_userid_password,
+    aws_api_gateway_documentation_part.users_userid_password_put,
+    aws_api_gateway_documentation_part.users_userid_reset-password,
+    aws_api_gateway_documentation_part.users_userid_reset-password_put,
+    aws_api_gateway_documentation_part.users_userid_send-verification,
+    aws_api_gateway_documentation_part.users_userid_send-verification_put,
+    aws_api_gateway_documentation_part.users_userid_lock,
+    aws_api_gateway_documentation_part.users_userid_lock_put,
+    aws_api_gateway_documentation_part.users_userid_unlock,
+    aws_api_gateway_documentation_part.users_userid_unlock_put
+  ]
+}
+
+# API
+
+resource "aws_api_gateway_documentation_part" "api" {
```

Comment:
Since you've already done all the work in creating the JSON schemas it'd be nice to have this stored as an OpenAPI file instead of Terraform resources so you can work on it locally. AWS does [support](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-documenting-api-quick-start-import-export.html) that, but not via Terraform it seems. Maybe one for later.